edge-util/test/Microsoft.Azure.Devices.Edge.Util.Test.Common/TempDirectory.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Util.Test.Common
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Text;
+
+    public class TempDirectory : IDisposable
+    {
+        List<string> dirs = new List<string>();
```

Comment:
why do you need to support list of temp dirs?  make it simple by supporting 1 temp dir at a time.  Create the directory in constructor and return TempDir when calling GetTempDir.  It is also confused that "GetTempDir" actually creates a directory.

Reply:
It's just used for testing. I want to get an empty folder each time, and not have to worry about cleaning it up. It does the same thing this rust library we use does: https://docs.rs/tempfile/3.1.0/tempfile/

Reply:
It's a list so the tests only have to create and dispose it once, but multiple tests can use it.